dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
It has two purposes - make the level readonly, and the answers viewable. I'm not sure of a great name that captures both. I'll def add a comment describing it.

This state is related to submitted, in that we shouldn't ever have viewable_answers be true while submitted is false. We discussed potentially making some sort of non-boolean column that captured all this, but (a) couldn't come up with a great concept that successfully captured everything and (b) Would then also need to manage rewriting a bunch of code that currently uses the submitted column.